sql/updates/world/6.x/2016_05_13_00_world.sql

```diff
@@ -0,0 +1,736 @@
+-- Tol Barad --
+
+-- Spawns are located half way down
+SET @CGUID := 250970; -- 214 creature entries
+SET @OGUID := 234355; -- 111 gameobject entries
```

Comment:
TrinityCore rev. 5f2c62392cad 2016-06-20 21:53:53 +0200 (6.x branch) (Win64, Release, Static), TDB 6.04 plus updates up to and including `2016_06_20_02_world.sql`. These values are available:
- 214 consecutive free GUIDs above 250000 in the `creature` table: 250970 - 251183
- 111 consecutive free GUIDs above 200000 in the `gameobject` table: 233475 - 233585

Looks like no new creatures are added since previous check, only gameobjects.